src/amo/components/Addon/index.js

```diff
@@ -178,10 +178,9 @@ export class AddonBase extends React.Component {
     const type = addon ? addon.type : ADDON_TYPE_EXTENSION;
 
     if (this.addonIsTheme()) {
-      let previewURL =
-        addon.previews.length > 0 && addon.previews[0].image_url
-          ? addon.previews[0].image_url
-          : null;
+      const previewHeader = addon.previews.length && addon.previews[0];
```

Comment:
It looks like this would set `previewHeader` to `false` if `addon.previews.length` is `0`, which would then cause the two statements below (e.g., `previewHeader.thumbnail_url`) to throw exceptions as `previewHeader` wouldn't be an object.

Reply:
hmm...actually if it comes across as `undefined`, I believe it should be assigned `null`. But I am going to revisit tho anyway to see about cleaning up :/